README.md

```diff
@@ -1,2 +1,45 @@
-# Hello-World
-my new repository
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello-World
+
+
+## Built With
+
+- HTML and CSS
+- VSCODE
+- GIT
+
+ 
+
+  
+
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better to remove unnecessary empty code lines to make your codebase cleaner.